upstairs/src/client.rs

```diff
@@ -571,9 +571,6 @@ impl DownstairsClient {
                 );
                 true
             }
-            DsState::Offline => {
-                panic!("can't deactivate while a downstairs is offline")
-            }
             s => {
```

Comment:
We no longer need this "protection" panic that was here before we supported replacing while Offline.